ios/Video/RCTVideo.m

```diff
@@ -747,12 +747,10 @@ - (void)attachListeners
 - (void)handleAVPlayerAccess:(NSNotification *)notification {
     AVPlayerItemAccessLog *accessLog = [((AVPlayerItem *)notification.object) accessLog];
     AVPlayerItemAccessLogEvent *lastEvent = accessLog.events.lastObject;
-    
-    /* TODO: get this working
```

Comment:
do you know what this TODO was about? Doesn't look like the code has changed much other than the rename of the prop

Reply:
ok I see why the name change, Video.js part was expecting onVideoBandwidthUpdate
https://github.com/react-native-community/react-native-video/blob/master/Video.js#L290

Reply:
@Komarev how the output data looks like on iOS? maybe it will be nice to add it on the readme as well, or is it outputting this same structure? https://github.com/react-native-community/react-native-video/blob/fa46e873069b1be4a1cbb7967ff5567c83b5e6b7/README.md#onbandwidthupdate

Reply:
Sorry about delay. Thats right, because of Video.js part was expecting 'onVideoBandwidthUpdate' name the feature didn't work correctly (I believe that's why it was commented out with TODO). But now it works as expected and according to readme documentation.